packages/@sparrow-workspaces/src/hoc/WithModal.svelte

```diff
@@ -15,7 +15,7 @@
 <Modal
   title={"Save Changes?"}
   type={"dark"}
-  width={"35%"}
+  width={"500px"}
```

Comment:
can we do it in percentage.

Reply:
I checked the percentage approach, but 40% modal looks little bit wide, and at 38%, the margins do not align well. I also looked at similar applications, and they seem to follow a standard pattern. If a 40% width works with the requirements and we’re okay with a wider look in full screen, we can go with it. Let me know what you think !!

![Screenshot from 2024-10-28 18-43-57](https://github.com/user-attachments/assets/ad2eee23-0de6-45d8-9c51-820422b5abec)
![Screenshot from 2024-10-28 18-44-43](https://github.com/user-attachments/assets/0ec7957b-6656-45ab-9f1b-29e1c1c4ca06)

Reply:
Will enhance it further, Looks good to me for now.
Good work 🎇